doc/JSON_FLAGS.md

```diff
@@ -853,6 +853,7 @@ Multiple death functions can be used. Not all combinations make sense.
 - ```CAN_DIG``` Can dig _and_ walk.
 - ```CAN_OPEN_DOORS``` Can open doors on its path.
 - ```CATFOOD``` Becomes friendly / tamed with cat food.
+- ```CANPLAY``` This creature can be played with if it's a pet.
```

Comment:
Please put these in alphabetic order.

Reply:
Done? I just edit the file in my fork and commit changes, right?

Reply:
Yep, this looks right